src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -388,69 +387,97 @@ public void dataQuality(){
                 addReplacement(replacements,"[dataorganisation]", dmp.getStructure());
             }
         }
+
+        if (dmp.getDataQuality() == null) {
+            addReplacement(replacements,"[dataqualitycontrol]", loadResourceService.loadVariableFromResource(prop, "dataQualityControl.no"));
+        }
+        else {
+            if (dmp.getDataQuality().isEmpty()) {
+                addReplacement(replacements,"[dataqualitycontrol]", loadResourceService.loadVariableFromResource(prop, "dataQualityControl.no"));
+            }
```

Comment:
could combine these and then return to reduce function complexity.
```java
        if (dmp.getDataQuality() == null || dmp.getDataQuality().isEmpty()) {
            addReplacement(replacements,"[dataqualitycontrol]", loadResourceService.loadVariableFromResource(prop, "dataQualityControl.no"));
            return;
        }
// continue with code from else branch without else (as we return in the if branch anyway)
```

Reply:
implemented as 
` if (dmp.getDataQuality() != null && !dmp.getDataQuality().isEmpty()) {`
combining null and isempty checks only works with &&, since if the first statement is false, it will skip the remaining statements.
connecting with || will execute the isEmpty check on a null value and throw a nullpointer.

Reply:
That is also fine :)

connecting with `||` will stop at the first statement evaluating to `true`, as the whole condition is already satisfied.
```java
        if (x == null || x.toString() == "") {
            System.out.println("null");
        }
```
would print "null"

Reply:
you are right, my bad :)